.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,93 @@
+---
+on: [push, pull_request]
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        scenario: ["TestMedusaDeploymentScenario/\"S3:\"", "TestMedusaDeploymentScenario/\"Minio:\"", "TestReaperDeploymentScenario"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-k8ssandra-buildx
+          restore-keys: |
+            ${{ runner.os }}-k8ssandra-buildx
+      
+      - name: Install Dependencies
+        run: |
+          set -x
+          sudo apt-get update || echo "something went slightly wrong with apt-get update"
+
+          # install Go
+          wget https://dl.google.com/go/go1.15.8.linux-amd64.tar.gz
+          sudo tar -C /usr/local -xzf go1.15.8.linux-amd64.tar.gz
+          export PATH=$PATH:/usr/local/go/bin
+          go version
```

Comment:
Have you tried to setup Go using the GitHub Action like they do in cass-operator?
https://github.com/datastax/cass-operator/blob/master/.github/workflows/kindIntegTest.yml#L32-L35

Either way works for me, but I guess the Action is _cleaner_

Reply:
Good point! I didn't think of searching for an action for this.
I'll update to use the action, it's cleaner indeed 👍

Reply:
done, thanks!